### PR TITLE
Update zyxel.md

### PR DESCRIPTION
**Pull request by pameruoso**
Addeding debrand guidance

### CODE REVIEW

**Review by simonebortolin — APPROVED** [review]

File: _sfp_cage/zyxel.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ layout: default
...
+
+1. access via ssh to the router with admin user (admin password is printed on the back of the router)
+2. Disable firmware version and modelcheck by running the following commands
+```
+zycli fwidcheck off
+zycli modelcheck off
+```
+3. You can close the ssh console, do not reboot the router
+4. Open the router web interface and in the maintenance/firmware upgrade section select the "Restore Default Settings After Firmware Upgrade" option.
+5. Select choose file to select the firmware file you want to upload and click Upload.
```
Comment (simonebortolin) [inline _sfp_cage/zyxel.md:31]: what firmware?

Comment (pameruoso) [reply _sfp_cage/zyxel.md:31]: I can provide my own compiled  firmware if it's ok to be published. I was thinking about adding it in the Miscellaneuos links

File: _sfp_cage/zyxel.md [hunk 2]
```diff
@@ -18,6 +18,32 @@ layout: default
...
+6. The router will automatically reboot and should get back up on 192.168.1.1
+
+# FW No-Brand Version V5.70(ACDZ.0)C0 compiled from Zyxel provided OpenSource package.
+The following is a firmware compiled starting from Zyxel provided OpenSource package. 
+
+--- Use at your own risk!!!! ---
+
+- [FW No-Brand Version V5.70(ACDZ.0)C0](https://mega.nz/file/7FZlWTzQ#F8Q_whoW4h1ETRUAzHe4PXIiK6TVj7uay_OtZxfmR6k)
```
Comment (ErnyTech) [inline _sfp_cage/zyxel.md:39]: The firmware is based on sources licensed under the GPL-2.0, GPL-3.0 and other more permissive licenses. Where are the sources of related to this firmware?

**Review by ErnyTech — CHANGES_REQUESTED** [review]

File: _sfp_cage/zyxel.md (hunk 2 above)
Comment (pameruoso) [reply _sfp_cage/zyxel.md:39]: Zyxel sent a link to me via email with a link to google drive to download the package. The original link only lasts 7 days.

Comment (ErnyTech) [reply _sfp_cage/zyxel.md:39]: You should publish sources for at least GPL-2.0, GPL-3.0 and other programs under copyleft licenses along with the firmware as required by the respective licenses

Comment (pameruoso) [reply _sfp_cage/zyxel.md:39]: Is it ok a link on my mega to host the original oss package?

Comment (ErnyTech) [reply _sfp_cage/zyxel.md:39]: It would be preferred on Github, Gitlab or similar but Mega is also legit to distribute the sources.

**Comment by simonebortolin** [comment]
@ErnyTech wouldn't it be better to put the link to your repo at this point?

**Comment by nanomad** [comment]
> @ErnyTech wouldn't it be better to put the link to your repo at this point?

I agree. We can put https://github.com/ErnyTech/zyxel-ex5601t0 as a link.

**Comment by simonebortolin** [comment]
Any news?

**Review by ErnyTech — APPROVED** [review]

**Pull request merged**